app/collections/Classrooms.coffee

```diff
@@ -0,0 +1,17 @@
+Classroom = require 'models/Classroom'
+CocoCollection = require 'collections/CocoCollection'
+
+module.exports = class Classrooms extends CocoCollection
+  model: Classroom
+  url: '/db/classroom'
+  
+  initialize: ->
+    @on 'sync', ->
+      for classroom in @models
+        classroom.capitalizeLanguageName()
```

Comment:
Odd... testing it, it looks like the binding does indeed work as-is. The documentation for `on` would leave me to believe you're correct, though. Weird.

http://backbonejs.org/#Events-on